lib/auth/init.go

```diff
@@ -355,7 +355,7 @@ func initCluster(ctx context.Context, cfg InitConfig, asrv *Server) error {
 		if _, err := asrv.UpsertRole(ctx, role); err != nil {
 			return trace.Wrap(err)
 		}
-		log.Infof("Created role: %v.", role)
+		log.Infof("Created role: %v.", role.String())
```

Comment:
These changes are not necessary. If the item implements `fmt.Stringer` then that will be invoked automatically.

Reply:
+1